### PR TITLE
Add remove

### PR DESCRIPTION
**Pull request by jlcarrascof**
**To Do List project**

In this milestone project, I

- Added different classes to the javascript file. Inlcuding the Todo list class, Store class and UI class.
- Created different methods that added , removed and store in local storage
- Created an object to instantiate the List with the following properties : index, desc, and completed
- Added a function for the text decoration if the user clicks on checkbox
- Added event listener to the icon button and the reset button
- Split the classes in different modules in order to import and export them

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,57 +1,97 @@
...
+    hr.remove();
+    li.remove();
+
+    // Reorder indexes
+    todoList = todoList.map((item, index) => {
+      item.index = index + 1;
+      return item;
+    });
+    localStorage.setItem('todoList', JSON.stringify(todoList));
+  };
```
Comment (levy002) [inline src/index.js:49]: - [ ] Process of removing a task seems to have a weird behavior after clicking the `dustbin` button for removing the task everything disappears and requires reloading in order to get the updated list. Kindly make sure to fix this error so your app will perform as expected.

  ### YOUR APP BEHAVIOUR
  chrome-capture-2023-6-14

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @jlcarrascof   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (jlcarrascof) [reply src/index.js:49]: Copied, sir ... I'll fix it 🙂👍

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @jlcarrascof  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Made changes as requested by the previous reviewer :+1: 
- App is now working as expected

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jlcarrascof** [comment]
Thanks a lot for your time, buddy Lucas @Lucas-Masaba 🙂🙂👍👍

**Pull request merged**